### PR TITLE
fix: checkout page bug when use 0 point

### PR DESCRIPTION
**Pull request by fandihungryhub**
https://3.basecamp.com/5190892/buckets/31126446/todos/6038769253#__recording_6057952187

### CODE REVIEW

File: stores/voucher.ts [hunk 1]
```diff
@@ -281,6 +281,23 @@ const useVoucherStore = defineStore("vouchers", {
         message: message,
       };
     },
+    useZeroPoint(points: number, oldPoints: number) {
+      const bookingChargeStore = useBookingChargeStore();
+      const defaultPrice =  bookingChargeStore.chargeObject.chargePrice + oldPoints;
```
Comment (afif-hh) [inline stores/voucher.ts:286]: kalau zero point hit API ulang aja mas

**Review by afif-hh — CHANGES_REQUESTED** [review]

File: stores/voucher.ts (hunk 1 above)
Comment (fandihungryhub) [reply stores/voucher.ts]: updated

**Comment by afif-hh** [comment]
is already tested ? @fandihungryhub

**Comment by fandihungryhub** [comment]
> is already tested ? @fandihungryhub

mas @afif-hh I have tested it with :

- signed in account **First Timer** which account has point
- signed in my account that doesn't have point
- guest account

**Review by afif-hh — APPROVED** [review]

**Pull request merged**